core/src/main/java/org/sonarsource/sonarlint/core/plugin/PluginClassLoaderDef.java

```diff
@@ -22,6 +22,7 @@
 import com.google.common.base.Preconditions;
 import com.google.common.base.Strings;
 import java.io.File;
+import java.net.URL;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.net.URL'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&pullRequest=165&issues=AWNE5hJhlRUrfRWxk5_-&open=AWNE5hJhlRUrfRWxk5_-)

[ISSUE_KEY]: AWNE5hJhlRUrfRWxk5_-